indico/modules/events/registration/models/items.py

```diff
@@ -345,6 +345,17 @@ def view_data(self):
     def is_active(self):
         return self.is_enabled and not self.is_deleted and self.parent.is_enabled and not self.parent.is_deleted
 
+    @property
+    def is_conditional(self):
```

Comment:
I think this should be named `is_condition`, since `is_conditional` would be more correct for the fields that references it.